### PR TITLE
Midterm Evaluation : Dataset-manipulation tools for simulated collider events

### PR DESCRIPTION
**Pull request by Blizzard57**

### CODE REVIEW

File: _gsocblogs/2022/blog_MCnet_KalpShah.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Implement additions to the program that allow different modes of operation for memory allocation. The possible modes being :
+    - Load the entire file into memory at once
+    - Sample events from the file and close it as per requirement
+- Change the working of the pile up files from cross sections to bias value ($\mu$)
+
+## Links
+
+The codes are dataset manipulation are present here : https://github.com/Blizzard57/data-manipulation
+
+The codes for pilemc are present here : https://gitlab.com/hepcedar/pilemc
```
Comment (agheata) [inline _gsocblogs/2022/blog_MCnet_KalpShah.md:52]: This repository only has the initial README.md commit. Please do not advertise empty repos.

File: _gsocblogs/2022/blog_MCnet_KalpShah.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+For pilemc, the tasks remaining are :
+- Complete the replication in python
+- Implement additions to the program that allow different modes of operation for memory allocation. The possible modes being :
+    - Load the entire file into memory at once
+    - Sample events from the file and close it as per requirement
+- Change the working of the pile up files from cross sections to bias value ($\mu$)
+
+## Links
+
+The codes are dataset manipulation are present here : https://github.com/Blizzard57/data-manipulation
```
[the remark this answers is not in the document]
Comment (agheata) [reply _gsocblogs/2022/blog_MCnet_KalpShah.md:50]: This repo is in your own space, with no review process visible. Is this planned to be merged in a project repository later on?

**Review by agheata — CHANGES_REQUESTED** [review]
Please address the open discussions, and ask your mentors to review your blog PR. Their OK as comment will do.

File: _gsocblogs/2022/blog_MCnet_KalpShah.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (agbuckley) [reply _gsocblogs/2022/blog_MCnet_KalpShah.md:50]: Yes, the intention in the second stage is for this to be merged to an MCnet repo on GitLab -- not yet decided whether to put it in a new repo, or make this part of MCUtils, I am leaning toward the latter

File: _gsocblogs/2022/blog_MCnet_KalpShah.md (hunk 1 above)
Comment (agbuckley) [reply _gsocblogs/2022/blog_MCnet_KalpShah.md:52]: @Blizzard57 You can link to the specific branch, https://gitlab.com/hepcedar/pilemc/-/tree/python-implementation but there is really no need for branching off an empty main so you may as well merge it now (and clean up the repo: things like the autotools cache files and `.vscode` dir shouldn't be committed.)

**Review by agheata — APPROVED** [review]

**Pull request merged**